### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by bemijonathan**
#### What does this PR do
Implementation of the desktop version of the home page of the osca website
#### Description of Task to be completed
A Responsive Home Page
#### How should this be manually tested
using different laptop screens and different browsers to check for errors in the layout.
#### What are the relevant issues this PR belongs to
#8 

#### Screenshots (Optional)
Screenshot from 2020-09-08 01-00-04

### CODE REVIEW

**Comment by Princesso** [comment]
Thanks @bemijonathan currently going through this PR

**Comment by Princesso** [comment]
Could you please fix the merge conflicts? @bemijonathan

File: src/pages/index.js [hunk 1]
```diff
@@ -5,15 +5,15 @@ import SEO from "../components/SEO/seo";
...
+// import Footer from "../components/footer";
 
 function IndexPage() {
   return (
     <Layout>
       <SEO keywords={["Open Source Community Africa", "OSCA", "Open Source Festival", "OSF"]} title="Home" />
       <Nav />
       <Header />
-      <Footer />
+      {/**<Footer /> **/}
```
Comment (Princesso) [inline src/pages/index.js]: Please can we totally discard this line of code since it is not used in our case?

File: src/pages/index.js [hunk 2]
```diff
@@ -5,15 +5,15 @@ import SEO from "../components/SEO/seo";
 
 import Nav from "../components/nav";
 import Header from "../components/header";
-import Footer from "../components/footer";
+// import Footer from "../components/footer";
```
Comment (Princesso) [inline src/pages/index.js]: Please can we totally discard this line of code since it is not used on this page?

File: src/components/nav.js [hunk 3]
```diff
@@ -40,29 +40,25 @@ function Nav() {
...
-              route: "#",
-              title: "Demo"
+              route: "#blog",
+              title: "Blog"
             }
           ].map(link => (
             <a
               className="block mt-0 mb-auto no-underline text-white text-xl lg:inline-block lg:-my-2 lg:ml-8 lg:text-sm uppercase"
-              key={link.title}
+              key={link.title + Math.random()}
```
Comment (Princesso) [inline src/components/nav.js]: Since titles are now unique is there a reason we still append the Math.random() method to the keys?

**Review by Princesso — COMMENTED** [review]
I also see package-lock.json as one of the files pushed, could you please git ignore package-lock.json? The reason is this: contributors can get a package lock by doing an npm install. That way everyone can manage package lock locally.

File: src/components/nav.js (hunk 3 above)
Comment (bemijonathan) [reply src/components/nav.js]: noted I will make the necessary changes

**Comment by Princesso** [comment]
Hello @bemijonathan, following on up this. I still see conflicting files  `src/components/header.js` and `src/components/nav.js` please could you rebase or resolve merge conflicts? Also is there an EDD on this PR?

**Comment by bemijonathan** [comment]
> Hello @bemijonathan, following on up this. I still see conflicting files `src/components/header.js` and `src/components/nav.js` please could you rebase or resolve merge conflicts? Also is there an EDD on this PR?

sorry, I have been a bit busy, I already started the mobile version will send a fix by tomorrow evening.

**Comment by Princesso** [comment]
> > Hello @bemijonathan, following on up this. I still see conflicting files `src/components/header.js` and `src/components/nav.js` please could you rebase or resolve merge conflicts? Also is there an EDD on this PR?
> 
> sorry, I have been a bit busy, I already started the mobile version will send a fix by tomorrow evening.

Great thank you !

**Comment by alabobriggs** [comment]
@Princesso are we good to go on this

**Comment by Princesso** [comment]
@alabobriggs merge conflicts are still not resolved on his end, so we are not

**Comment by bemijonathan** [comment]
can you resolve the conflict on your end because I can even see the conflict on my end.

**Pull request closed without merging**